standard/ERCs/erc-6900.md

```diff
@@ -683,6 +683,12 @@ The `executeFromModule` function MUST allow modules to call execution functions
 
 The `executeFromModuleExternal` function MUST allow modules to call external addresses as specified by its parameters on behalf of the modular account. If the calling module's manifest did not explicitly allow the external call within `permittedExternalCalls` at the time of installation, execution MUST revert.
 
+### Extension
+
+#### Semi-modular account
+
+Account implementers may opt to design a semi-modular account, where certain features, such as default validation, are integrated into the core account. This approach should ensure compatibility with fully modular accounts, maintaining interoperability across different implementations.
```

Comment:
Nit: we should probably capitalize "should" as "SHOULD", as the top part of the spec states.

Reply:
MAY as well :)